sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientLiveTests.cs

```diff
@@ -44,8 +44,19 @@ public async Task StartTrainingCanAuthenticateWithTokenCredential()
             var trainingFilesUri = new Uri(TestEnvironment.BlobContainerSasUrl);
 
             TrainingOperation operation = await client.StartTrainingAsync(trainingFilesUri, useTrainingLabels: false);
+            CustomFormModel model = null;
```

Comment:
In this file we're just adding try/finally blocks to make sure we always delete models we create for testing.